core/settings.py

```diff
@@ -125,3 +125,11 @@
 
 # session
 SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+
+# Not actually used - Just required for LiveServerTestCase
+DATABASES = {
```

Comment:
During deployment, will this necessitate the creation of an SQLite3 database? Had  similar problem when i used sessions & database

Reply:
Nope, this is in-memory